apps/test/unit/componentLibrary/LinkTest.jsx

```diff
@@ -1,246 +1,85 @@
-//         label="Checkbox label"
-//         checked={checked}
-//         onChange={onChange}
-//         indeterminate={indeterminate}
-//       />
-//     );
-//
-//     checkbox = screen.getByDisplayValue('test-checkbox');
-//
-//     expect(spyOnChange).to.have.been.calledTwice;
-//     expect(spyOnChange).to.have.been.calledWith(false);
-//     expect(checkbox.checked).to.be.false;
-//     expect(checkbox.disabled).to.be.false;
-//     expect(checkbox.indeterminate).to.be.false;
-//   });
-//
-//   it("Checkbox - renders disabled checkbox, doesn't change on click", async () => {
-//     const user = userEvent.setup();
-//     const spyOnChange = sinon.spy();
-//
-//     let checked = false;
-//     const onChange = () => {
-//       checked = !checked;
-//       spyOnChange(checked);
-//     };
-//
-//     // Initial render
-//     const {rerender} = render(
-//       <Checkbox
-//         name="test-checkbox"
-//         value="test-checkbox"
-//         label="Checkbox label"
-//         checked={checked}
-//         onChange={onChange}
-//         disabled={true}
-//       />
-//     );
-//
-//     let checkbox = screen.getByDisplayValue('test-checkbox');
-//     expect(checkbox).to.exist;
-//
-//     expect(checkbox.checked).to.be.false;
-//     expect(checkbox.disabled).to.be.true;
-//     expect(checkbox.indeterminate).to.be.false;
-//
-//     await user.click(checkbox);
-//
-//     // Re-render after user's first click
-//     rerender(
-//       <Checkbox
-//         name="test-checkbox"
-//         value="test-checkbox"
-//         label="Checkbox label"
-//         checked={checked}
-//         onChange={onChange}
-//         disabled={true}
-//       />
-//     );
-//
-//     checkbox = screen.getByDisplayValue('test-checkbox');
-//
-//     expect(spyOnChange).to.not.have.been.called;
-//     expect(checkbox.checked).to.be.false;
-//     expect(checkbox.disabled).to.be.true;
-//     expect(checkbox.indeterminate).to.be.false;
-//
-//     await user.click(checkbox);
-//
-//     // Re-render after user's second click
-//     rerender(
-//       <Checkbox
-//         name="test-checkbox"
-//         value="test-checkbox"
-//         label="Checkbox label"
-//         checked={checked}
-//         onChange={onChange}
-//         disabled={true}
-//       />
-//     );
-//
-//     expect(spyOnChange).to.not.have.been.called;
-//     expect(checkbox.checked).to.be.false;
-//     expect(checkbox.disabled).to.be.true;
-//     expect(checkbox.indeterminate).to.be.false;
-//   });
-// });
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import sinon from 'sinon';
+
+import {expect} from '../../util/reconfiguredChai';
+
+import Link from '@cdo/apps/componentLibrary/link';
+
+describe('Design System - Link', () => {
+  it('Link - renders with correct text', () => {
+    render(<Link href="https://studio.code.org/home">Home</Link>);
```

Comment:
nit: constant for `"https://studio.code.org/home"` just to ensure the test is referencing the same value? also can be reused across multiple tests